=== modified file 'src/daemon/dbus/Makefile.am'
--- a/src/daemon/dbus/Makefile.am	2015-02-13 14:27:42 +0000
+++ b/src/daemon/dbus/Makefile.am	2015-03-27 13:07:49 +0000
@@ -100,8 +100,6 @@
    $(NULL)
 
 dbusservicedir = $(DBUS_SERVICES_DIR)
-dbusservice_DATA = \
-    $(SERVICES_DIR)/com.google.code.AccountsSSO.gSingleSignOn.service
 nodist_dbusservice_DATA = \
     $(SERVICES_DIR)/com.google.code.AccountsSSO.gSingleSignOn.service
 

=== modified file 'src/gplugind/main.c'
--- a/src/gplugind/main.c	2015-02-16 15:35:04 +0000
+++ b/src/gplugind/main.c	2015-03-27 12:41:18 +0000
@@ -126,7 +126,7 @@
 
 static void _list_watch_path()
 {
-    g_print(_plugin_path());
+    g_print("%s", _plugin_path());
 }
 
 int main (int argc, char **argv)

