Index: gsignond.conf.in
===================================================================
--- gsignond.conf.in	2013-10-24 11:50:05.510126880 +0200
+++ gsignond.conf.in	2013-10-24 16:14:34.583555094 +0200
@@ -10,7 +10,7 @@
 # Define extension / platform adaptation module to use.
 # 'default' is a dummy one with no encrypted storage, but with basic Unix
 # discretionary access control.
-#Extension = default
+Extension = pantheon
 #
 # Base path where user specific database directories will be located.
 @BASE_STORAGE_DIR@

